packages/os/early-boot-config.service

```
@@ -14,4 +14,4 @@ RemainAfterExit=true
 StandardError=journal+console
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=preconfigured.target
```

Previously this unit was effectively `RequiredBy: configured.target`.

```suggestion
RequiredBy=preconfigured.target
```

---

Updated to `RequiredBy`